src/kakarot/accounts/eoa/library.cairo

```diff
@@ -145,6 +159,13 @@ namespace ExternallyOwnedAccount {
         );
         memcpy(response, return_data, return_data_len);
 
+        // See Argent account
+        // https://github.com/argentlabs/argent-contracts-starknet/blob/c6d3ee5e05f0f4b8a5c707b4094446c3bc822427/contracts/account/ArgentAccount.cairo#L132
+        // Using here the EVM msg_hash instead of the Starknet tx.hash
+        transaction_executed.emit(
+            msg_hash=msg_hash, response_len=return_data_len, response=return_data, success=success
```

Comment:
is there a particular reason why we choose the msg_hash?
we don't want to compute the tx hash (might be too expensive to do a second keccak)?

Reply:
the msg_hash is enough to unambiguously map the event to the initial tx, so in the current situation I see no reason to do any extra computation